ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -753,7 +751,7 @@ export default defineComponent({
         localState.isValidAllocation &&
         !localState.hasGroupsWithNoOwners &&
         (isTransferDueToSaleOrGift.value ? !TransSaleOrGift.hasMixedOwners.value : true) &&
-        (isTransferToSurvivingJointTenant.value ? localState.isValidDeathCertificate : true) &&
```

Comment:
We can't use `isValidDeathCertificate` prop here, as it is always false when going back from Review & Confirm page, and it won't recompute without making changes in DeathCertificate component.